contents/handbook/growth/sales/crm.md

```diff
@@ -81,28 +81,52 @@ You can also just manually add a user to HubSpot under 'Contacts'. When creating
+|                | 21-50                                                                                                                                                                                                 | 3     |
+|                | 51-100                                                                                                                                                                                                | 4     |
+|                | 101-1000                                                                                                                                                                                              | 5     |
+|                | 1000+                                                                                                                                                                                                 | 3     |
+| Ability to pay | Revenue >= $10m or Raised > $20m                                                                                                                                                                      | 5     |
+|                | Revenue $1m-$10m or Raised > $10m                                                                                                                                                                     | 4     |
+| Role           | engineering                                                                                                                                                                                           | 5     |
+|                | leadership                                                                                                                                                                                            | 4     |
+|                | product                                                                                                                                                                                               | 3     |
+|                | other (known)                                                                                                                                                                                         | 1     |
+| Customer type  | B2B                                                                                                                                                                                                   | 5     |
+|                | B2C                                                                                                                                                                                                   | 2     |
+| Country        | Not in Austria, Belgium, Brazil, Canada, Denmark, Finland, France, Germany, Iceland, Ireland, Italy, Japan, Netherlands, Norway, Portugal, Spain, Sweden, Switzerland, United Kingdom, United States  | -7    |
```

Comment:
Would we consider any of the following in our target? (These are just suggestions, so feel free to ignore, just seems like we might be being a bit _too_ restrictive with this list!)

- Australia
- New Zealand
- Mexico (latam tech hub)
- Singapore (far east tech hub)
- South Korea
- Any other EU countries like Estonia, Poland
- UAE (middle east tech hub)
- Israel (middle east tech hub)

Reply:
(The reason I want to get specific with this btw is because it affects where we then target our content and ads - if we think Israel will result in low quality signups for example, we'll make sure we're not running ads there)

Reply:
I'd say yes to all of the above except for UAE and Mexico as (from prior experience) their propensity to pay for software is low.  I'll make the update in the docs to reflect that change though.